Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponents.cs

```diff
@@ -174,7 +177,7 @@ private static void AttachEvents(VB.VBComponents components)
         {
             lock (Locker)
             {
-                if (_components == null)
+                if (_components == null && components != null)
```

Comment:
Note: this works, but only because the `SafeComWrapper` base class overrides equality/inequality operators.